include/effects/strip/fireeffect.h

```diff
@@ -199,16 +199,23 @@ class FireEffect : public LEDStripEffect
 
         for (int i = 0; i < LEDCount; i++)
         {
+            auto sum = 0;

```

Comment:
Now uses the average of the pixels in the group, rather than just the first value, and looks MUCH more realistic.